### ISSUE TITLE
Add KISAO and TEDDY and COMBINE archive info

### ISSUE THREAD

**Issue by matthiaskoenig** [SED-ML/sed-ml.github.io#7]
Currently these concepts are not well documented/linked on the homepage.

* Ontologies Used Kinetic Simulation Algorithm Ontology (https://bioportal.bioontology.org/ontologies/KISAO)
    * https://www.ebi.ac.uk/ols/ontologies/kisao
* https://www.ebi.ac.uk/ols/ontologies/teddy
    * Terminology for Description of Dynamics

**Comment by jonrkarr** [comment]
KiSAO is described in the specifications. I think this is sufficient explanation.

One thing that would be helpful is to indicate the simulation type(s) and algorithm(s) involved in each example archive. This is helpful for quickly finding examples that are relevant to a particular type of simulation.

Why does TEDDY need to be mentioned?

**Comment by nickerso** [comment]
The idea was to use TEDDY to describe expected behaviour from a given SED-ML; i.e., so that you can search for experiments that are expected to produce particular dynamic characteristics. I'm not sure if anyone or any tool is actually doing this yet. And probably better nowadays to look more at annotating simulation results archived in a database such as biosimulators?

**Comment by jonrkarr** [comment]
I agree annotation of simulation results is a big hole in SED-ML. Given that TEDDY or other possible resources aren't yet used, I don't think they need to be mentioned on the website at this point. They should be documented when there is a scheme for using them.

RunBioSimulations includes database of simulation results. As we fully migrate to HDF5, we will have the flexibility to store a variety of metadata. TEDDY could be used. Annotations could be embedded into HDF5 files or annotations could be kept in a separate file linked via metaids which are saved into HDF5 files. HDF5 isn't XML, but otherwise we could make this work fairly similarly to how OMEX meta works with SBML and CellML.
